src/App.jsx

```diff
@@ -29,13 +30,32 @@ function App() {
         throw new Error(`Error: ${response.status} `)
       }
       const data = await response.json()
+
       console.log(data);
+
+      // const todos = data.records.map(record => ({
```

Comment:
Nit: remove commented code

Reply:
the same code is present at line number 53 to 56